### ISSUE TITLE
New changes in python-rq is breaking django-rq

### ISSUE THREAD

**Issue by ayys** [rq/django-rq#421]
With the recent changes in python-rq, they have removed `UnpickleError` from rq/exceptions.py in this [commit](https://github.com/rq/rq/commit/e1cbc3736c55d7a72be121a170d1b8c811619dc2#diff-27817b1a9be843cacb4ebeb20393c486L22)

So now when I install django-rq, I get the following error
```
  File "/root/.local/share/virtualenvs/src-VBU3rafc/lib/python3.7/site-packages/django_rq/urls.py", line 3, in <module>
    from . import views
  File "/root/.local/share/virtualenvs/src-VBU3rafc/lib/python3.7/site-packages/django_rq/views.py", line 12, in <module>
    from rq.exceptions import NoSuchJobError, UnpickleError
ImportError: cannot import name 'UnpickleError' from 'rq.exceptions' (/root/.local/share/virtualenvs/src-VBU3rafc/lib/python3.7/site-packages/rq/exceptions.py)
```

**Comment by davidtoulmin** [comment]
I'm experiencing the same issue. I've just installed django-rq, and trying to run the server I'm getting the error
```
File "[home and project directory]/myenv/lib/python3.6/site-packages/django_rq/templatetags/django_rq.py", line 4, in <module>
    from rq.exceptions import UnpickleError
ImportError: cannot import name 'UnpickleError'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
[more error message]
django.template.library.InvalidTemplateLibrary: Invalid template library specified. ImportError raised when trying to load 'django_rq.templatetags.django_rq': cannot import name 'UnpickleError'
```

**Comment by wowi42** [comment]
Just force rq to 1.2.2

```
pip install rq==1.2.2
```

**Comment by wowi42** [comment]
it seems that 1.3.0 is working too

**Comment by awbacker** [comment]
Yes, pin it in requirements to 1.3.0
```
django-rq == 2.3.1
rq == 1.3.0
```

**Comment by ayys** [comment]
I checked with 1.3.0, seems to be working for me too

**Comment by steffann** [comment]
That's a workaround, but as a user of `django-rq` I'd prefer not to have to manage dependencies of dependencies. It would be good if `django-rq` would specify `rq<1.4.0` itself :)

**Comment by adamghill** [comment]
Looks like this is fixed in django-rq==2.3.2 (https://github.com/rq/django-rq/commit/9f7dbe32623b085ea85799fc20dc0fb36b9903ab).
